erpnext/compliance/taxes.py

```diff
@@ -15,6 +15,9 @@
 
 
 def calculate_cannabis_tax(doc):
+	if frappe.get_cached_value("Stock Settings", None, "ignore_licenses_for_tax_calculation"):
```

Comment:
@hrwX for single doctype use frappe.db.get_single_value

Reply:
- The value is fetched from cache and not from db.